test/jdk/java/awt/MenuItem/GiantFontTest.java

```diff
@@ -40,10 +41,17 @@ public class GiantFontTest {
     public static void main(String[] args) throws Exception {
         String INSTRUCTIONS = """
                         A frame with one menu will appear.
-                        On Solaris, the menu's font should be quite large (48 point).
+                        On Linux, the menu's (present on menu bar) font should
+                        be quite large (48 point).
                         If not, test fails.
 
-                        On Windows, the font should be normal size.  If so, test passes.
+                        On Windows, the menu's (present on menu bar) font
+                        should be normal size.
```

Comment:
I believe, “If the menu text is clipped by the title bar, or is painted over the title bar or client area…” belongs here.

(I forgot to click *Add comment* button.)

Reply:
Moved the test instructions.